### PR TITLE
UIREQ-940 create JEST/RTL test cases for DraggableRow.js

### PR DESCRIPTION
**Pull request by ssandupatla**
UIREQ-940 create JEST/RTL test cases for DraggableRow.js
URL: https://issues.folio.org/browse/UIREQ-940

### CODE REVIEW

**Review by cchitneedi — APPROVED** [review]

File: src/components/SortableList/DraggableRow.test.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    additionalClasses: ['extra-class'],
+  };
+  const provided = {
+    innerRef: jest.fn(),
+    draggableProps: {
+      style: { transform: 'translate(0, 0)' },
+      'aria-describedby': 'row-1',
+    },
+    dragHandleProps: {},
+  };
```
Comment (Dmitriy-Litvinenko) [inline src/components/SortableList/DraggableRow.test.js:17]: Could you please add 
 ```
 const props = {
    cells,
    provided,
    snapshot: {
      isDragging: false,
    },
    rowClass: 'row-class',
    rowIndex: 1,
    rowProps,
  };
```
  ?

File: src/components/SortableList/DraggableRow.test.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  test('renders DraggableRow correctly', () => {
+    render(
+      <DraggableRow
```
Comment (Dmitriy-Litvinenko) [inline src/components/SortableList/DraggableRow.test.js:20]: Could you please change to `<DraggableRow {...props} />`?

File: src/components/SortableList/DraggableRow.test.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        rowIndex={1}
+        rowProps={rowProps}
+      />
+    );
+    expect(screen.getByText('Cell 1')).toBeInTheDocument();
+  });
+
+  test('If isDragging prop false, class should be extra-class row-class', () => {
+    render(
+      <DraggableRow
```
Comment (Dmitriy-Litvinenko) [inline src/components/SortableList/DraggableRow.test.js:34]: Could you please change to `<DraggableRow {...props} />`?

File: src/components/SortableList/DraggableRow.test.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        rowIndex={1}
+        rowProps={rowProps}
+      />
+    );
+    expect(document.getElementById('ModuleContainer')).toBeInTheDocument();
+  });
+
+  test('If isDragging prop True, class should be extra-class row-class DraggableRow', () => {
+    render(
+      <DraggableRow
```
Comment (Dmitriy-Litvinenko) [inline src/components/SortableList/DraggableRow.test.js:63]: Could you please change to 
```
    const draggableProps = {
      ...props,
      snapshot: {
        isDragging: true,
      },
    };
```
`<DraggableRow {...draggableProps} />`?

**Review by Dmitriy-Litvinenko — CHANGES_REQUESTED** [review]

**Comment by Dmitriy-Litvinenko** [comment]
Could you please fix feedbacks?

**Comment by Dmitriy-Litvinenko** [comment]
Could you please fix feedbacks?

File: src/components/SortableList/DraggableRow.test.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  });
+
+  const draggableProps = {
+    ...props,
+    snapshot: {
+      isDragging: true,
+    },
+  };
+
+  test('DraggableRow using portal', () => {
```
[the remark this answers is not in the document]
Comment (ssandupatla) [reply src/components/SortableList/DraggableRow.test.js:46]: @Dmitriy-Litvinenko - Changes done as per your suggestion.

**Review by Dmitriy-Litvinenko — APPROVED** [review]

**Pull request merged**